lega/conf/defaults.ini

```diff
@@ -34,15 +36,9 @@ s3_chunk_size = 4194304
 
 ## Connecting to Local Broker
 [broker]
-enable_ssl = no
-host = ega_mq
-port = 5672
-username = guest
-password = guest
-vhost = /
-connection_attempts = 10
-retry_delay = 10
-heartbeat = 0
+connection = amqps://guest:guest@mq:5672/%2F?heartbeat=0&connection_attempts=30&retry_delay=10
+verify_peer = no
```

Comment:
`verify_peer` needs to be true as MQ is started with:
```
ssl_options.verify = verify_peer
ssl_options.fail_if_no_peer_cert = true
```

Reply:
Or I can make that configurable since I already have a couple of other changes that needs to go into the MQ image.

Reply:
Done.